docs/source/notebooks/wps_decompose_flow_vectors_rename.ipynb

```diff
@@ -0,0 +1,196 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# wps_decompose_flow_vectors\n",
+    "\n",
+    "#### wps_decompose_flow_vectors is a process that runs the [decompose_flow_vectors](https://github.com/pacificclimate/climate-explorer-data-prep#decompose_flow_vectors-create-normalized-unit-vector-fields-from-a-vic-routing-file) module of PCIC Climate Explorer Data Preparation Tools. Here, the client will try to connect to a remote osprey instance using the url parameter."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from birdy import WPSClient\n",
+    "from pkg_resources import resource_filename\n",
+    "import requests\n",
+    "import os\n",
+    "from datetime import datetime\n",
+    "from wps_tools.utils import copy_http_content\n",
+    "from netCDF4 import Dataset\n",
+    "from tempfile import NamedTemporaryFile\n",
+    "import math\n",
+    "\n",
+    "# Ensure we are in the working directory with access to the data\n",
+    "while os.path.basename(os.getcwd()) != \"thunderbird\":\n",
+    "    os.chdir('../')\n",
+    "\n",
+    "url = 'https://docker-dev03.pcic.uvic.ca/twitcher/ows/proxy/thunderbird/wps'\n",
```

Comment:
Is there some reason you are using this instead of `http://localhost:5001/wps`? Should I be using this for the generate climos demo as well?

Reply:
The objective of birds is to run the processes in remote Docker containers instead of running in a manually set up local virtual environment. The URL directs to the `thunderbird` container in PCIC's docker server, which hosts various birds.

Reply:
Ultimately it is setup such that we can use a local instance if one exists, otherwise use the docker instance. We will have to come back and add this feature in the future.